docs/content/reference/exploiters/SMBExec.md

```diff
@@ -6,4 +6,4 @@ tags: ["exploit", "windows"]
 ---
 ### Description
 
-This exploit brute forces machines using credentials provided by the user (see [configuration]({{< ref "/usage/configuration" >}}) for instructions) and hashes gathered from infected systems by Mimikatz.
+This exploit brute forces machines using credentials provided by the user (see [configuration](/usage/configuration) for instructions) and hashes gathered from infected systems by Mimikatz.
```

Comment:
This links to the high-level configuration page, wheras the elsewhere we link directly to the credentials page. Should this link to the credentials page instead?